libs/crypt.js

```diff
@@ -0,0 +1,33 @@
+const CryptoJS = require('crypto-js');
+
+const characters = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
+
+function randomString(length) {
+  var result = '';
+  for (var i = length; i > 0; --i) result += characters[Math.floor(Math.random() * characters.length)];
+  return result;
+}
+
+const crypt = {
+  aes(salt) {
+    return {
+      encrypt(text) {
+        return CryptoJS.AES.encrypt(text, salt).toString();
+      },
+      decrypt(text) {
+        const bytes = CryptoJS.AES.decrypt(text, salt);
+        return bytes.toString(CryptoJS.enc.Utf8);
+      }
+    };
+  }
+};
+
+crypt.salt = function (salt) {
```

Comment:
function needs comment 'Using AES256 bit encryption' and it might be worth include a link to https://cryptojs.gitbook.io/docs/#pbkdf2 in case people want to assess it?

Reply:
Code is not annotated, see https://github.com/Fliplet/fliplet-agent/blob/master/libs/crypt.js

Reply:
I meant now*, not "not" :)